subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -26,6 +26,10 @@
 /* Convert from ms to 0.625ms units */
 #define ADV_INT_FAST_MS    20
 
+#ifndef CONFIG_BT_MESH_RELAY_ADV_SETS
+#define CONFIG_BT_MESH_RELAY_ADV_SETS 0
```

Comment:
Minor. I remember that not everyone is happy when `CONFIG_...`-type macros are redefined.

Reply:
I believe this is a common pattern for when the kconfig symbol is blocked by a `depends on`, but we still need it around to build the stack. See for instance https://github.com/zephyrproject-rtos/zephyr/blob/e2edb3e405b6e8158bab068736af2e916d157ebc/subsys/logging/log_backend_rtt.c#L14-L34

Reply:
I had the opposite experience 😄 
https://github.com/zephyrproject-rtos/zephyr/pull/33477#discussion_r599478755